rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -220,6 +221,10 @@ cat > /var/www/rutorrent/conf/config.php <<EOL
 \$log_file = '${RU_LOG_FILE}';
 \$do_diagnostic = ${RU_DO_DIAGNOSTIC};
 
+// If true, skip checking for remote exes.
+// Set to true if rTorrent is hosted on the SAME machine as ruTorrent.
+\$localHostedMode = ${RU_LOCAL_HOSTED_MODE};
```

Comment:
Just set the same as https://github.com/Novik/ruTorrent/blob/21b6a27ea1a97b56720d7e05c3eb378103cef77a/conf/config.php#L62 and enforce to true.

```suggestion
\$localHostedMode = true; 		// Set to true if rTorrent is hosted on the SAME machine as ruTorrent
```

No need to have an env var for this.